docs/src/main/tut/docs/combinators.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: docs
+title: Combinators
```

Comment:
Nitpick: It feels like this page is specific to `ConfigReader` combinators and thus the title is excessively general.   Feel free to ignore.

Reply:
I understand what you mean, but it's not only the title of this section. I explained the reasoning for focusing on `ConfigReader` on the PR description:

> Unlike formats for information interchange like JSON, the most common use case for PureConfig is arguably the reading of configs to Scala data structures, with the writing part being very uncommon by comparison. In order to make the documentation clearer I updated all pages to refer only to readers and added a dedicated "Config Writers" section explaining the writing part;

I assumed that people will use PureConfig mostly for reading configurations. The dedicated "Config Writers" section mentions that `ConfigWriter` also has combinators available. Do you think that it isn't enough?